daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actors-usage.md

```diff
@@ -136,8 +136,11 @@ public void ConfigureServices(IServiceCollection services)
     // Register actor runtime with DI
     services.AddActors(options =>
     {
-        // Register actor types and configure actor settings
+        // Register actor types and configure actor settings.
```

Comment:
Just a rule, I don't consider periods necessary on comments - more often than not, comments reflect partial sentence fragments and not full sentences due appropriate capitalization and punctuation.

Reply:
Removed the period.